documents/charter.md

```diff
@@ -15,17 +15,26 @@ bitrate termed "application limit"—to the endpoint originating the UDP
 In the context of this charter, a "network device" is defined as anything on 
 the path of a UDP 5-tuple that is capable of dropping or delaying packets.
 
+## Open Issues for Investigation
+
+The working group will investigate and determine the necessity of a "client
+conformance signaling" mechanism, that allows clients to signal their receipt
+of the application limits and ability to conform to them. This investigation
```

Comment:
I feel the term "application limits" could be misinterpreted as meaning "the limits for this particular application", and hence risk pushback from national regulators who are concerned with applications being treated differently by networks (i.e. net neutrality). I think the network rather signals "throughput limits", based on the user connection state,  which an application can decide to account for.

Reply:
@Kevsy  the charter currently defines "application limit" in the following way, so that we can use the term throughout the text:
"This WG aims to establish a mechanism for network devices intending to rate-limit a UDP 5-tuple to communicate an upper bound on achievable bitrate termed "application limit"—to the endpoint originating the UDP 5-tuple."

Do you think this definition is problematic for the same reasons?

Reply:
> Do you think this definition is problematic for the same reasons?
I do :)  - I understand the intention, as shorthand for 'given what the network knows about the application endpoint, this is the upper bound on bitrate'. But as well as my point above on regulatory concerns, the wording '**application** limit' does not include context that applies to all applications, namely network congestion and end-user context (especially relevant on cellular). Hence my preference for "throughput limit" encompasses all the factors.

Reply:
@Kevsy yeah you make a good point. The reason we landed on "application limit" instead of throughput limit is because the terminology we had before which was simply "maximum achievable throughput" caused all sorts of confusion and made lots of people thing we were replacing congestion control. "Application limit" is meant to convey that this work is suggesting making flows more "application-limited", e.g. by affecting the ABR decisions, rather than strictly affecting transport decisions.

I do not immediately have a great suggestion for how to thread the needle here between these two issues.

Reply:
@Kevsy what do you think of this PR otherwise, if you did a find/replace in your head to change that particular language? Are you supportive of the idea of punting the client signaling issue to the WG?

I think we can have another issue/discussion addressing the terminology.

Reply:
@mjoras sure, I think the client signaling question can be resolved in the WG. So I'm otherwise fine with the PR and can raise the terminology separately as you say.